activerecord/lib/active_record/connection_adapters/postgresql/oid/cidr.rb

```diff
@@ -28,6 +28,10 @@ def serialize(value)
             end
           end
 
+          def changed?(old_value, new_value, _new_value_before_type_cast)
+            new_value != old_value || new_value.prefix != old_value.prefix
```

Comment:
```suggestion
            super || new_value.prefix != old_value.prefix
```

Reply:
I think using `super` makes Cidr#changed? requirement clear. https://github.com/rails/rails/blob/734719d8af033062cb7088a1353be6276659e3c1/activemodel/lib/active_model/type/value.rb#L84-L86

Reply:
I just prefer this to keep the argument order inside the method. old_value, then new_value.
```ruby
super || old_value.prefix != new_value.prefix
```

Reply:
Thank you for your advices.
It has fixed to adopt `super` and keeping the argument order.